packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {mkdtemp, rmdir, writeFile} from 'fs/promises'
+
+const pkg = require('../../package.json')
+import * as upload2 from './upload2'
```

Comment:
@olibrook meta question - why are we moving from `upload` => `upload2` is this to avoid customers confusing the two if they've customized pwa-kit-dev or something?

Reply:
Actually is `upload2` even a file or is this outdated? I don't see it in the changeset

<img width="299" alt="image" src="https://user-images.githubusercontent.com/47546998/214936423-14999c16-761d-4f24-a688-d292be871690.png">

Reply:
Oopsie, I renamed the file to something less stupid and missed this occurence. The correct new name is `script-utils`, I will fix it in the test! thxxx